client/src/items/Computer.ts

```diff
@@ -0,0 +1,66 @@
+import { ItemType } from '../../../types/Items'
+import store from '../stores'
+import Item from './Item'
+import Network from '../services/Network'
+import { openComputerDialog } from '../stores/ComputerStore'
+
+export default class Computer extends Item {
+  id?: string
+  currentUsers = new Array<string>()
+
+  constructor(scene: Phaser.Scene, x: number, y: number, texture: string, frame?: string | number) {
+    super(scene, x, y, texture, frame)
+
+    this.itemType = ItemType.COMPUTER
+    this.currentUsers = []
```

Comment:
This is redundant, you already have currentUsers = new Array<string>() above.

Reply:
Removed